content/manager/delete-blueprint.md

```diff
@@ -8,11 +8,18 @@ weight: 800
 
 Deleting a blueprint removes its model from the database and deletes its resources from the fileserver. Deleting a blueprint does not delete the deployments created from that blueprint or resources of those deployments.
 
-To delete a blueprint from the CLI, run
+To delete a blueprint run the following command.
 
 {{< gsHighlight  bash >}}
 cfy blueprints delete [OPTIONS] BLUEPRINT_ID
 {{< /gsHighlight >}}
 
```

Comment:
Move the options to before the UI instructions:

The delete options are:

* -v, --verbose - Show verbose output. You can supply this up to three times, for example `-vvv`.
* -t, --tenant-name - Specify the tenant where the blueprint in stored. (Default: current tenant)